source/frontend/src/components/layout/SideNavBar/styles.ts

```diff
@@ -20,7 +20,7 @@ export const SideNavBar = styled.div`
   align-items: center;
   grid-area: iconbar;
   background-color: ${props => props.theme.tenant.colour};
-  transition: 0.5s width;
+  transition: 0.3s width;
```

Comment:
what is the motivation for this change?

Reply:
I was trying to reduce the animation time, but I see that it's not needed.